1-exercises/D-do-while/exercise.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function evenNumbersSum(n) {
-    // TODO
+    let arr = [], i = 0;
+    do {
+        if (i % 2 !== 0) {
+            i++;
+            continue;
+        } else {
+            arr.push(i);
+            i += 2;
+        }
+    } while (arr.length < n);
+    return arr.reduce((a, b) => a + b);
```

Comment:
Same consideration as above, can you try implementing this algorithm without using 'continue'?

Reply:
This will work, but you don't need to repeat the increment in the two code branches. You can achieve the same effect this way:
```
function evenNumbersSum(n) {
    let arr = [], i = 0;
    do {
        if (i % 2 === 0) {
            arr.push(i);
        }
        i += 2;
    } while (arr.length < n);
    return arr.reduce((a, b) => a + b);
}
```

Reply:
@gixo oh yeah. That is cleaner.